src/main/flight/pid.c

```diff
@@ -266,9 +269,15 @@ void pidInitFilters(const pidProfile_t *pidProfile)
         ptermYawLowpassApplyFn = (filterApplyFnPtr)pt1FilterApply;
         pt1FilterInit(&ptermYawLowpass, pt1FilterGain(pidProfile->yaw_lowpass_hz, dT));
     }
+
+    float RC = 1.0f / ( 2.0f * M_PIf * pidProfile->throttle_boost_cutoff );
```

Comment:
Technically this is RC lowpass,  pt1FilterGain is available (except for uint8_t parameter)